### PR TITLE
docs: add category use and more maintainer

### PR DESCRIPTION
**Pull request by Scrum**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,6 +16,19 @@ All HTML transformations are made by plugins. And these plugins are just small p
 
 For more detailed information about PostHTML in general take a look at the [docs][docs-url]
 
+## Us Use
```
Comment (anikethsaha) [inline README.md]:
```suggestion
## Who's Using Posthtml ?
```

What do you think

**Review by anikethsaha — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -33,6 +46,12 @@ For more detailed information about PostHTML in general take a look at the [docs
       <br />
       <a href="https://github.com/voischev">Ivan Voischev</a>
     </td>
+     <td align="center">
+      <img width="150 height="150"
+      src="https://github.com/anikethsaha.png?size=150">
+      <br />
+      <a href="https://github.com/anikethsaha">Anix Anikethsaha</a>
```
Comment (anikethsaha) [inline README.md]:
```suggestion
      <a href="https://github.com/anikethsaha">Aniketh Saha</a>
```

Its a bit confusing have different username and name  😄

Comment (Scrum) [reply README.md]: Sorry

Comment (anikethsaha) [reply README.md]: no issue, I should have a single username 👍 . its confusing for others as well.

**Review by anikethsaha — APPROVED** [review]

**Review by cossssmin — APPROVED** [review]

**Pull request merged**